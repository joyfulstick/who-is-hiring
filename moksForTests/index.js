export const STORIES = [
  19591871,
  19589520,
  19588812,
  19587763,
  19586072,
  19579227,
  19578070,
  19574209,
  19570596,
  19567427,
]

export const JOBS = [
  {
    by: 'linaz',
    id: 19591871,
    score: 1,
    time: 1554571793,
    title: 'Jerry (YC S17) Is Hiring Senior Software Developer (Toronto)',
    type: 'job',
    url: 'https://www.workable.com/j/089F60DE31',
  },
  {
    by: 'aharm',
    id: 19589520,
    score: 1,
    time: 1554530862,
    title: 'Iris Automation Is Hiring a Computer Vision Expert â€“ AI for Drones',
    type: 'job',
    url: 'http://www.irisonboard.com/careers/',
  },

  {
    by: 'danperito',
    id: 19588812,
    score: 1,
    time: 1554518120,
    title: 'Faire (YC W17) Is Looking to Hire Senior Data Scientists',
    type: 'job',
    url:
      'https://boards.greenhouse.io/indigofair/jobs/4005276002?gh_jid=4005276002',
  },
  {
    by: 'joshbuckley',
    id: 19587763,
    score: 1,
    time: 1554504547,
    title:
      'Mino Games (YC W11) Is Hiring a Senior Game Engineer in Montreal, QC',
    type: 'job',
    url: 'https://mino-games.workable.com/j/7AC7233C2B?viewed=true',
  },
  {
    by: 'sytse',
    id: 19586072,
    score: 1,
    time: 1554492082,
    title: 'Gitlab (YC W15, All-remote) is hiring 17 back end engineers',
    type: 'job',
    url:
      'https://about.gitlab.com/jobs/apply/backend-engineer-x-17-4055697002/',
  },
  {
    by: 'alooPotato',
    id: 19579227,
    score: 1,
    time: 1554432513,
    title: 'Streak (YC S11) Is Hiring Kotlin Engineers in Vancouver',
    type: 'job',
    url: 'https://www.streak.com/careers/android-lead-vancouver',
  },
  {
    by: 'thedogeye',
    id: 19578070,
    score: 1,
    time: 1554419770,
    title: 'Flexport is hiring software engineers in Chicago',
    type: 'job',
    url: 'https://www.flexport.com/careers/department/engineering',
  },
  {
    by: 'ttruett',
    id: 19574209,
    score: 1,
    time: 1554395390,
    title: 'Ambition (YC W14) Hiring Engineer with Django and React Experience',
    type: 'job',
    url: 'https://ambition.com/career/opportunity/full-stack-engineer/',
  },
  {
    by: 'jon_dahl',
    id: 19570596,
    score: 1,
    time: 1554372129,
    title:
      'Mux is growing quickly and hiring engineers to work on analytics at scale',
    type: 'job',
    url: 'https://mux.com/jobs/',
  },
  {
    by: 'roger_lee',
    id: 19567427,
    score: 1,
    time: 1554327248,
    title:
      'Human Interest (401k for small businesses) is hiring a Technical Product Manager',
    type: 'job',
    url: 'https://humaninterest.com/careers',
  },
]
