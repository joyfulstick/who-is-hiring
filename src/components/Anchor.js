import React from 'react'
import AnchorStyle from '../styles/AnchorStyle'

const Anchor = ({ href }) => (
  <AnchorStyle href={href} target="_blank" rel="noopener noreferrer">
    {href}
  </AnchorStyle>
)

export default Anchor
