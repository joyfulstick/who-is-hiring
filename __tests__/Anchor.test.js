import { shallow } from 'enzyme'
import React from 'react'
import { JOBS } from '../moksForTests'
import Anchor from '../src/components/Anchor'

describe('<Anchor />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Anchor href={JOBS[0].url} />)
    expect(wrapper).toMatchSnapshot()
  })
})
