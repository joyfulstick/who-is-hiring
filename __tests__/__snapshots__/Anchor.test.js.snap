// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Anchor /> renders correctly 1`] = `
<styled.a
  href="https://www.workable.com/j/089F60DE31"
  rel="noopener noreferrer"
  target="_blank"
>
  https://www.workable.com/j/089F60DE31
</styled.a>
`;
